### DISCUSSION TITLE
Hot Reload Not Working with Vite on a Clean Orchid Installation

### DISCUSSION THREAD

**Discussion by SharkJohny** [orchidsoftware/platform#2790] [General]
How can I resolve an issue where hot reload is not working with Vite? On a clean Laravel installation with Vite, everything works fine, but on a clean Orchid installation, hot reload doesn't seem to function. Any suggestions or solutions would be greatly appreciated.

**Comment by czernika** [comment]
Hard to say without any provided configuration. My guess Vite doesn't watch changes in Orchid directory. Try to extend default Laravel's refresh paths with Orchid directory

```js
import laravel, { refreshPaths } from 'laravel-vite-plugin'

export default defineConfig({
    plugins: [
        laravel({
            input: [
                // entrypoints
            ],
            refresh: [
                ...refreshPaths,
                'app/Orchid/**',
            ],
        }),
    ],
})
```

P.S. You may add [conditional guard](https://vitejs.dev/guide/api-hmr.html#required-conditional-guard) in order for HMR to work
